apps/cowswap-frontend/src/common/pure/PendingOrderNotification/PendingOrderNotification.test.ts

```diff
@@ -0,0 +1,140 @@
+import { WRAPPED_NATIVE_CURRENCIES as WETH } from '@cowprotocol/common-const'
+import { OrderKind, SupportedChainId } from '@cowprotocol/cow-sdk'
+import { ToastMessageType } from '@cowprotocol/events'
+import { CurrencyAmount, Token, ChainId } from '@uniswap/sdk-core'
+
+import { UiOrderType } from 'utils/orderUtils/getUiOrderType'
+
+import { PendingOrderNotificationProps, getPendingOrderNotificationToast } from '.'
+
+const WETH_MAINNET = new Token(ChainId.MAINNET, WETH[1].address, 18, 'WETH', 'Wrapped Ether')
+const DAI_MAINNET = new Token(
+  ChainId.MAINNET,
+  '0x6b175474e89094c44da98b954eedeac495271d0f',
+  18,
+  'DAI',
+  'Dai Stablecoin'
+)
+
+const wethAmount = CurrencyAmount.fromRawAmount(WETH_MAINNET, 1e18)
+const daiAmount = CurrencyAmount.fromRawAmount(DAI_MAINNET, 2000 * 1e18)
```

Comment:
Can you add to the amounts (or a different test if you prefer) a decimal amount, like 1.231 ETH or something

Reply:
<img width="435" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/a50423cf-163d-4985-bd39-9aea036e1040">

done a6026f029c1cf5795ea0ff55d58fb5804ee0e1ee